docs/troubleshooting.md

```diff
@@ -1,13 +1,11 @@
 # Troubleshooting
 
 ## Problems with Kangal installation
-
 To troubleshoot Kangal you will need access to your Kubernetes cluster.
 
 - Log in to the cluster where Kangal is installed
 - Check if Kangal Proxy and Controller pods are up and running
 - Check logs of Kangal Proxy and Kangal Controller
 
 ## Problems with a specific load test
-
-You can make basic troubleshooting using Kangal API endpoints.
+You can make basic troubleshooting using Kangal API endpoints or either exploring load test Pods if is the case of your backend.
```

Comment:
You don't really need to exec into the pods because we have /logs endpoint to get the same logs as you can get by `kubectl logs master-pod`

Reply:
Yes, but I want to make sure that people will understand that `proxy` is a optional component

Reply:
On the other hand, we created proxy to provide a possibility to operate with load tests without having access to Kubernetes cluster at all. That makes it possible for quality engineers and even managers who don't have cluster permissions, to get all the info about load test and create/delete it with proxy. 

There is a borderline between `installing` Kangal to the cluster (should be done by engineers/DevOps, who have cluster access) and `using` Kangal by proxy (which is supposed to be not a DevOps work). I agree that we need installation steps, but it should be the part of developer documentation, but not user documentation. And user docs should not require any kubectl commands.

Reply:
I wouldn’t assume a scenario, could be that quality engineers has access to cluster or even a fully automated test.
Troubleshooting doc is not separated in developer/user, I prefer to list all options and let the reader decides which one works best for him/her.